jaeger_client/sampler.py

```diff
@@ -449,10 +448,10 @@ def _update_adaptive_sampler(self, per_operation_strategies):
 
     def _update_rate_limiting_or_probabilistic_sampler(self, response):
         s_type = response.get(STRATEGY_TYPE_STR)
-        if s_type == SamplingManager.SamplingStrategyType.PROBABILISTIC:
+        if s_type == PROBABILISTIC_SAMPLING_STRATEGY:
```

Comment:
we could also do `|| s_type == SamplingManager.SamplingStrategyType.PROBABILISTIC`